pkg/operation/seed/seed.go

```diff
@@ -728,6 +741,7 @@ func BootstrapCluster(k8sSeedClient kubernetes.Interface, seed *Seed, secrets ma
 		"ingress": map[string]interface{}{
 			"basicAuthSecret": monitoringBasicAuth,
 		},
+		"cluster-identity": map[string]interface{}{"clusterIdentity": &seed.Info.Status.ClusterIdentity},
```

Comment:
```suggestion
		"clusterIdentity": &seed.Info.Status.ClusterIdentity,
```
I don't see `cluster-identity` in the seed charts above, or am I missing something?

Reply:
ok, forget about this, this is ok, there seems to be a minor issue in `values.yaml` of the `seed-bootstrap` chart that confused me